scripts/cangen/README.md

```diff
@@ -0,0 +1,41 @@
+# Cangen
+Cangen is a cli package used to generate C++/C files from a corresponding dbc file. This tool automates the can generation process and ensures can generation is consistent between all projects. 
```

Comment:
Remove the start of this sentence. We do not need to state that cangen is a package; that is obvious. The reader is interested in what cangen does. This sentence should read like

"Generate C++ code from a CAN DBC file. Ensures repeatable generation between all projects."